addons/project/models/project_task_recurrence.py

```diff
@@ -252,7 +253,21 @@ def _create_next_task(self):
             new_task = self.env['project.task'].sudo().create(create_values)
             recurrence._create_subtasks(task, new_task, depth=3)
 
-    def _set_next_recurrence_date(self):
+    def _set_next_recurrence_date(self, date_start=None):
```

Comment:
It is not really stable. 😕 

If someone does a customisation to override that method and add another parameter, it could crash his customisation because we give a date_start as additional parameter. 😕